packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -640,10 +640,9 @@ class AndroidGradleBuilder implements AndroidBuilder {
     required Directory outputDirectory,
     required String buildNumber,
   }) async {
-
     final FlutterManifest manifest = project.manifest;
-    if (!manifest.isModule && !manifest.isPlugin) {
-      throwToolExit('AARs can only be built for plugin or module projects.');
+    if (!manifest.isModule) {
+      throwToolExit('AARs can only be built for module projects.');
```

Comment:
is this related?

Reply:
No, this was brought in from your [commit](https://github.com/flutter/flutter/pull/141019/commits/d2bd4441c0c6330a6729c0a5c37b3a68a4efdb50) actually. There was a change you made that my CP was depending on. I thought originally that I would need to only merge previously committed PRs for a CP.

It's probably best to revert the changes from your commit and just make the necessary changes for this PR to work right?